.github/workflows/create-release.yaml

```diff
@@ -118,8 +114,7 @@ jobs:
 
   create-draft:
     name: Create draft release
-#    needs: bump-sec-scanners-config
-    needs: validate-release
+    needs: [bump-sec-scanners-config,validate-release]
```

Comment:
In this job we need to use the most recent commit ref created in `bump-sec-scanners-config`. Example of how to do this https://github.com/kyma-project/btp-manager/blob/dc810239ffd4dc92c52ba38235f23194a8b41f0c/.github/workflows/create-release.yaml#L244C30-L244C30